### ISSUE TITLE
Bad choices-box position when input has a positioned ancestor

### ISSUE THREAD

**Issue by marianobianchi** [yourlabs/jquery-autocomplete-light#7]
I was having a problem using autocomplete inside a jquery-ui dialog: the box with choices that appears when typing inside the autocomplete input was bad positioned. 

Checking how CSS position property works i found this [[1]](http://www.w3schools.com/cssref/pr_class_position.asp):

> Value = absolute: The element is positioned relative to its first positioned (not static) ancestor element

So, if the input has a positioned (not static) ancestor, the absolute position is relative to the position of the ancestor.

Here is a simple example of how this fail (the choices URL is missing):

```
<html>
  <head>
    <meta charset="UTF-8">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
    <script src="jquery.autocomplete-light.min.js"></script>
    <style>
      .yourlabs-autocomplete {
        background-color: #FFF;
      }
    </style>
  </head>
  <body style="background-color: #3FA0FF;">
    <div id="data" style="width:500px; height: 200px; background-color: #55BFFF; position: absolute; top: 50px; left: 100px;">
      <form>
        <label for="username">Username</label>
        <input id="username" type="text" />
      </form>
    </div>

    <script>
      $('#username').yourlabsAutocomplete({
          url: '<URL>',
          choiceSelector: '.choice',
      }).input.bind('selectChoice', function(e, choice, autocomplete) {
          alert('Picked option: ' + choice.text());
      });
    </script>
  </body>
</html>
```

When running this with a simple and valid choices URL, we will see a page like this:
![jquery_autocomplete_bad_positioning](https://cloud.githubusercontent.com/assets/1700075/9281521/fdc93f2a-429b-11e5-8cc4-6c879fad335d.png)

The problem is that `yourlabs.Autocomplete.prototype.fixPosition` method assumes input doesn't have a positioned (not static) ancestor.

[1]: http://www.w3schools.com/cssref/pr_class_position.asp

**Comment by jpic** [comment]
I know, but it's the only way to support django's default admin theme properly for now, here's the bug that made our positioning method change: https://github.com/yourlabs/django-autocomplete-light/issues/265 so DAL 2.2.0 introduced this change. I don't like it so much and people have been complaining about it, but it supports django default admin theme. We'd really like to do something better about it, could you make a pull request ?

Thanks !

**Comment by jpic** [comment]
Isn't this fixed by #8 ? (GG BTW B))

**Comment by marianobianchi** [comment]
Yes! Did i open two issues for the same problem? Or an issue is opened when i open a pull request?
